### ISSUE TITLE
False positive when not exporting a component from a file

### ISSUE THREAD

**Issue by nandita1** [ArnaudBarre/eslint-plugin-react-refresh#111]
If there is any component in the file, it is expected that atleast one component should be exported which is not the desired behavior

Eg:
```
const Abc = () => <></>;

export const abc = () => {
  return [];
};
```
I get a lint error saying
`Fast refresh only works when a file has exports. Move your component(s) to a separate file.eslint(react-refresh/only-export-components)` but this is a valid scenario as some of the utils may use react components internally.

If I export a component instead of the util, then there is no error.

**Comment by ArnaudBarre** [comment]
This is not a false positive. The component `Abc` is not able to be refreshed in isolation in that case. If `abc` is imported in an entry point, changing the code of the component will trigger a full page reload, which would not happen if the component was in it's own file.

As any lint rule, disabling it locally when you feel the tradeoff is not good is totally expected.
